### PR TITLE
39 ramiz

### PR DESCRIPTION
**Pull request by RamizAlasgarov**
2 time chache readme.md

### CODE REVIEW

File: HackerRankIssues/README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 This folder contains tasks for HackerRank platform (https://www.hackerrank.com/)
 We used issues from "1 Week Preparation Kit"
 JavaScript first
+Java second
+slojniye izmeneniya
```
Comment (rasnaut) [inline HackerRankIssues/README.md:6]: Please, write in English

File: HackerRankIssues/README.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 This folder contains tasks for HackerRank platform (https://www.hackerrank.com/)
 We used issues from "1 Week Preparation Kit"
 JavaScript first
+Java second
```
Comment (rasnaut) [inline HackerRankIssues/README.md:5]: That's good

**Review by adragin — APPROVED** [review]
Cool changes!

**Review by rasnaut — CHANGES_REQUESTED** [review]
Please change code according to comments

File: HackerRankIssues/README.md (hunk 1 above)
Comment (RamizAlasgarov) [reply HackerRankIssues/README.md:6]: ok

**Review by rasnaut — APPROVED** [review]

**Pull request merged**